packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -0,0 +1,403 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+    :route="backRoute"
+    :icon="icon"
+  >
+    <KPageContainer v-if="device" :style="pageHeight">
+      <KGrid gutter="48" class="edit-sync-schedule">
+
+        <KGridItem>
+          <h1>{{ $tr('editSyncScheduleSubTitle') }}</h1>
+        </KGridItem>
+
+        <KGridItem>
+          <p>{{ device.device_name }} </p>
+        </KGridItem>
+
+        <KGrid class="align-kselects">
+          <KGrid>
+            <KGridItem>
+              <KSelect
+                v-model="selectedItem"
```

Comment:
One of these KSelects is still giving a lot of validator warnings, so we should work out which one(s) and clean up the inputs/options.